cli/lsp/language_server.rs

```diff
@@ -1229,6 +1230,9 @@ impl Inner {
     if let Err(err) = self.update_cache().await {
       self.client.show_message(MessageType::WARNING, err);
     }
+    if let Ok(deno_dir) = DenoDir::new(self.maybe_global_cache_path.clone()) {
+      init_log_file(deno_dir.lsp_log_file_path());
+    };
```

Comment:
I suggest we write it in the CWD the program is run with - something like `deno_lsp_<iso_date>.txt`